libs/ui/src/lib/flags/flags.component.html

```diff
@@ -12,9 +13,10 @@
   <mat-divider></mat-divider>
 
   <mat-accordion [multi]="true" displayMode="flat">
-    <mat-expansion-panel *ngFor="let fieldGroup of fieldGroups" [expanded]="fieldGroup.expanded" (opened)="resizeFlags.emit()" (closed)="resizeFlags.emit()">
+    <mat-expansion-panel *ngFor="let fieldGroup of fieldGroups" [expanded]="fieldGroup.expanded" (opened)="resizeFlags.emit()"
+      (closed)="resizeFlags.emit()">
       <mat-expansion-panel-header>
-        <mat-panel-title *ngIf="fieldGroups.length > 1" [ngSwitch]="fieldGroup.type" [ngClass]="'js-group-' + fieldGroup.type">
```

Comment:
This class was used to drive e2e tests. They are commented out for now since they seem to be causing memory issues in CI, but something to keep in mind.